parquet/src/test/java/org/apache/iceberg/avro/TestParquetReadProjection.java

```diff
@@ -32,7 +32,7 @@ public class TestParquetReadProjection extends TestReadProjection {
   protected GenericData.Record writeAndRead(
       String desc, Schema writeSchema, Schema readSchema, GenericData.Record record)
       throws IOException {
-    File file = temp.resolve(desc + ".parquet").toFile();
```

Comment:
This was modified by https://github.com/apache/iceberg/pull/8056. 
Hence, reverting this line and making it compatible with parent class in `iceberg-core`

Reply:
It would be better to first switch `TestReadProjection` from `iceberg-core` to JUnit5 and then removing `TestReadProjection` from `iceberg-parquet`

Reply:
That may require moving to `Junit5` for a whole `iceberg-core` module. I am afraid I don't have a time to work on that. 
@coded9, if you are intersted, please feel free to work on it.  

I was doing a POC of partition stats where I had to move all the files of iceberg-parquet into iceberg-core, that time I found this duplicate code.

Reply:
Done. Rebased. Now it is on Junit 5.